please

```diff
@@ -20,6 +20,13 @@ if [ -f /.dockerenv ]; then
   . /etc/nix/profile.sh;
 fi
 
+# Detect OSX
+SHASUM="sha256sum"
+uname -a | grep -q Darwin
+if [ $? -eq 0 ] ; then
```

Comment:
`if uname -a | grep -q Darwin; then` should also work

Reply:
:+1: looks like this fixed taskcluster